src/plugins/newsfeed/public/components/flyout_list.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useCallback, useContext } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiIcon,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiLink,
+  EuiFlyoutFooter,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiText,
+  EuiBadge,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiHeaderAlert } from '../../../../legacy/core_plugins/newsfeed/public/np_ready/components/header_alert/header_alert';
+import { NewsfeedContext } from './newsfeed_header_nav_button';
+import { NewsfeedItem } from '../../types';
+
+export const NewsfeedFlyout = () => {
+  const { newsfeed, kibanaVersion, setFlyoutVisible } = useContext(NewsfeedContext);
+  const closeFlyout = useCallback(() => setFlyoutVisible(false), [setFlyoutVisible]);
+
+  return (
+    <EuiFlyout
+      onClose={closeFlyout}
+      size="s"
+      aria-labelledby="flyoutSmallTitle"
+      className="kbnNews__flyout"
+    >
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="s">
+          <h2 id="flyoutSmallTitle">
+            <FormattedMessage
+              id="newsfeed.components.flyoutList.flyoutTitle"
+              defaultMessage="What's new"
+            />
+          </h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody className={'kbnNews__flyoutAlerts'}>
+        {newsfeed.map((item: NewsfeedItem) => {
+          return (
+            <EuiHeaderAlert
+              key={item.hash}
+              title={item.title}
+              text={item.description}
+              action={
+                <EuiLink target="_blank" href={item.linkUrl}>
```

Comment:
This might be ok for Phase 1, but it looks like at some point, news items can link internally in Kibana, like take you to a new section of Management, etc. In those cases, we would want to open in the same window.

Reply:
Side note, there is also a new `external` prop on `EuiLink` that adds the popout icon.

Reply:
@YulNaumenko let's use that`external` prop for this. It should fix an issue I saw where there is no space before the popout link: `https://user-images.githubusercontent.com/908371/67977309-1460ba00-fbd5-11e9-9c37-c9016614b85d.png`